dipy/tracking/tests/test_utils.py

```diff
@@ -73,6 +74,28 @@ def test_density_map():
     assert_array_equal(dm, expected)
 
 
+def test_to_voxel_coordinates_precision():
+    # To simplify tests, use an identity affine. This would be the result of
+    # a call to _mapping_to_voxel with another identity affine.
+    transfo = np.array([[1.0, 0.0, 0.0],
+                        [0.0, 1.0, 0.0],
+                        [0.0, 0.0, 1.0]])
+
+    # Offset is computed by _mapping_to_voxel. With a 1x1x1 dataset
+    # having no translation, the offset is half the voxel size, i.e. 0.5.
+    offset = np.array([0.5, 0.5, 0.5])
+
+    # Without the added tolerance in _to_voxel_coordinates, this streamline
+    # should raise an Error in the call to _to_voxel_coordinates.
```

Comment:
Oh - sorry - I see that makes no sense.

All good. I'll wait for travis to run this and then merge. 

Again - thanks for filling that in!